core/new-gui/src/app/workspace/component/workspace.component.ts

```diff
@@ -5,17 +5,23 @@ import { OperatorPanelComponent } from './operator-panel/operator-panel.componen
 import { PropertyEditorComponent } from './property-editor/property-editor.component';
 import { WorkflowEditorComponent } from './workflow-editor/workflow-editor.component';
 import { ResultPanelComponent } from './result-panel/result-panel.component';
+import { OperatorMetadataService } from '../service/operator-metadata/operator-metadata.service';
 
 
 
 @Component({
   selector: 'texera-workspace',
   templateUrl: './workspace.component.html',
-  styleUrls: ['./workspace.component.scss']
+  styleUrls: ['./workspace.component.scss'],
```

Comment:
Shouldn't there be changes in workspace html too where you include operator label and operator panel html tags? Or are you planning to get it reviewed at the last?

Reply:
operator panel component is already included in workspace.component.html, operator label component is included inside operator-panel.component.html